Source/WPF/MyMoney/View Selectors/BalanceControl.xaml

```diff
@@ -66,8 +66,8 @@ For example, you can use this to swap out a reconciled transaction for another o
                                     <ColumnDefinition Width="Auto"/>
                                     <ColumnDefinition Width="*"/>
                                 </Grid.ColumnDefinitions>
-                                <Button x:Name="PreviousBalanceHelp" Click="PreviousBalanceHelp_Click">
-                                    <Image Source="/MyMoney;component/Icons/Help.png" />
+                                <Button x:Name="PreviousBalanceHelp" Click="PreviousBalanceHelp_Click" Background="Transparent">
+                                    <Label>ℹ️</Label>

```

Comment:
Doesn't scream "click here for more information" to me:
![image](https://user-images.githubusercontent.com/3336415/208190947-1835b003-e5b0-4da7-a322-96c9d08c08a1.png)

FontSize 16 is a bit better...
![image](https://user-images.githubusercontent.com/3336415/208191039-57550249-668d-489e-8463-65877aa8a2aa.png)

Reply:
Is there a way to get a blue background like I see here?
https://emojipedia.org/information/

Reply:
Waddya think about this version?
```
              <Button x:Name="CurrentDeltaHelp" Click="CurrentDeltaHelp_Click" Background="#3A86C2" Padding="0,0,0,1">
                <Label FontSize="18" Foreground="White" HorizontalAlignment="Center" VerticalAlignment="Center">ℹ️</Label>
              </Button>
```
![image](https://user-images.githubusercontent.com/3336415/208197470-e64c6a3d-d5e2-4f57-aeea-3636c2b74eaa.png)

Reply:
![image](https://user-images.githubusercontent.com/3336415/208197587-de8498a8-c967-4ef9-a197-a0f0da465303.png)

Reply:
I got the colors from Win+. emoji selector:

![image](https://user-images.githubusercontent.com/3336415/208197697-caff7aac-ee0a-448c-901e-951eb3e09d6e.png)

and we could templatize this button style on our themes so we can use it anywhere.

Reply:
I'd prefer to get rid of the white border which seems to be coming from a unicode variation 0xFE0F, but I couldn't figure out how... Notice github shows the windows emoji like this: ℹ️

Reply:
This all looks good the  way you are doing now. ship it